packages/ag-charts-community/src/scene/shape/line.ts

```diff
@@ -28,6 +28,16 @@ export class Line extends Shape {
     @SceneChangeDetection({ redraw: RedrawType.MAJOR })
     y2: number = 0;
 
+    set x(value: number) {
+        this.x1 = value;
+        this.x2 = value;
+    }
+
+    set y(value: number) {
+        this.y1 = value;
+        this.y2 = value;
+    }
```

Comment:
This addition to the `Line` interface seems confusing (as it already has `x1`, `x2`, `y1` and `y2` properties)?

Reply:
this is actually following d3 convention
it's a utility setter to use in case both points on the same axis are of identical values
it helps turn this:
```
element.x1 = 123;
element.x2 = 123;
```
into:
```
element.x = 123;
```
I think it helps cleanup the code and it actsw as an alternative to chaining, such as `element.x1 = element.x2 = 123;`

Reply:
I'm not 100% convinced (especially if we also have `setProperties()` to improve readability), but let's roll with it.